### PR TITLE
Contact form validation

### PR DESCRIPTION
**Pull request by tsheporamantso**
**In this milestone, I have:-**

- [ ] Used client-side validation to catch and throw errors in the UI.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,3 +11,49 @@ document.querySelectorAll('.navbar-tools').forEach((n) => n
...
+
+function checkInputs(event) {
+  // get the value from inputs
+  const emailValue = email.value;
+
+  if (isEmail(emailValue)) {
+    return true;
+  }
+
+  // setErrorFor('Invalid Email, it should only contain lowercase');
```
Comment (omarsalem7) [inline index.js]: - [ ] Kindly remove the commented code from the codebase to make your code clean and follow JavaScript best practices.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @tsheporamantso 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️

- [ ] Kindly fix the broken MIT license link in the README file and add your own license.
- [ ] _Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (tsheporamantso) [reply index.js]: Done, Thank you.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: APPROVED ♻️♻️
Hi @tsheporamantso, 

To highlight 👍🏽 
✔️ There are no commented codes
✔️ There are no linter errors

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] As this is your last review, I would make this optional. Kindly ensure the license in your project is fixed like the previous reviewer requested. You can check [this out](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) for more guidelines on how to fix it.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request closed without merging**